python/src/FieldFunctionalChaosSobolIndices_doc.i.in

```diff
@@ -1,6 +1,10 @@
 %feature("docstring") OT::FieldFunctionalChaosSobolIndices
 "Sobol indices from a functional decomposition.
 
+.. warning::
```

Comment:
the warning box are huge, probably needs some css tweaking

Reply:
Not sure what to do about this. Does it need to be fixed before the release? It is not too big an issue IMO.

Reply:
Fixed.